src/database/sql/audio.py

```diff
@@ -456,7 +490,8 @@ def get_peak(self, audio_id: str) -> str:
         query = """
             SELECT peak, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.